sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -18,8 +20,13 @@
 from azure.core.exceptions import ServiceRequestError
 from ._generated.models import OCIManifest
 
+from azure.mgmt.containerregistry import ContainerRegistryManagementClient
+from azure.mgmt.containerregistry.models import ImportImageParameters, ImportSource, ImportMode
+from azure.identity import DefaultAzureCredential, AzureAuthorityHosts, ClientSecretCredential
+from azure.identity.aio import DefaultAzureCredential as AsyncDefaultAzureCredential
+
 if TYPE_CHECKING:
-    from typing import List, Dict, IO
+    from typing import Dict, IO
```

Comment:
These imports should be moved to line 13

Reply:
I have a separate PR for hint style and will get rid of TYPE_CHECKING, similar to one you have reviewed for ApppConfig. 
Update: the PR is in draft, will be ready for review after the mypy PR get merged.